BACKEND/userAuth/src/main/java/com/apptime/auth/model/TaskReport.java

```diff
@@ -0,0 +1,85 @@
+package com.apptime.auth.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * @author Qi Zhang
+ * This is the PO for TaskReport
+ * Use Case: TMGP4-26, TMGP4-31
+ */
+@Entity
+public class TaskReport {
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @OneToOne
```

Comment:
Why we are creating this relationship to task. the report is just  information that we extract from tasks. 
this creates a update /delete cascade problem.